Providers/Profile.swift

```diff
@@ -2,8 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import Alamofire
-import Foundation
```

Comment:
You should probably add a warning to future travellers that adding extra imports here may have deleterious effects on push notifications.